Lib/delay.h

```diff
@@ -56,6 +57,11 @@ static inline uint32_t millis(void)
 	return systick_millis_count; // single aligned 32 bit is atomic;
 }
 
+static inline uint32_t microsToTicks(uint32_t) __attribute__((always_inline, unused));
+static inline uint32_t microsToTicks(uint32_t usec)
+{
+    return usec * (F_CPU / 1000000);
```

Comment:
Please use tabs instead of spaces. In general my rule of thumb is "copy the formatting of the surrounding code". This includes things like spaces around ()'s and {}'s.